### ISSUE TITLE
What is the relationship between Sekando scripts and this?

### ISSUE THREAD

**Issue by sonniesedge** [sekando/feednami-client#6]
There is a script listed [at Sekando](https://toolkit.sekando.com/docs/en/feednami) that is called "[feednami-client 1.1](https://static.sekandocdn.net/static/feednami/feednami-client-v1.1.js)". 

However, the only version mentioned here is [v1.0.2](https://cdn.rawgit.com/richardkazuomiller/feednami-client/master/releases/1.0.2.min.js).

Are these in sync? Are they even related?

**Comment by richardkazuomiller** [comment]
Sorry about the confusion.

Feednami v1.1 is part of Sekando Web Toolkit and my plan is to have users create an account and authenticate API requests using the script specified in its documentation. The API is mostly identical except for authentication, but I don't plan on updating the client script in this repository.

When I released Feednami last year, I did so without any authentication system or plan of how to fund development or costs of having fast servers in multiple regions. I tried to fix that after the fact in a few different ways, including patronage and having people create a paid Sekando account after they decided to use it in production, however since authentication wasn't required to make requests, people either didn't read that part of the documentation or considered the monthly fee optional. I'm currently in the process of addressing that by moving Feednami and another project which will be a paid service that I've been working on into Sekando Web Toolkit.

People who are currently using either version of the API today don't need to worry about the monthly fee if the cost is prohibitively expensive. I don't want to break anyone's website so I'll whitelist the hostnames of websites which start using Feednami before a specific date (probably a few weeks from now).
